dask/utils.py

```diff
@@ -1272,6 +1272,62 @@ def format_time(n):
     return "%.2f us" % (n * 1e6)
 
 
+def format_time_ago(n):
+    """Calculate a '3 hours ago' type string from a Python datetime.
+
+    >>> from datetime import datetime, timedelta
```

Comment:
```suggestion

    Examples
    --------
    >>> from datetime import datetime, timedelta
```

Reply:
Can I also ask for a Parameters section with a type of `n` (or perhaps use Python type hints there?)

Maybe also add newlines in the example below in between the output of one section and the `past = ` line of the next?